src/main/java/me/zeroeightsix/kami/util/combat/SurroundUtils.kt

```diff
@@ -59,6 +62,36 @@ object SurroundUtils {
         return block == Blocks.BEDROCK || block == Blocks.OBSIDIAN || block == Blocks.ENDER_CHEST || block == Blocks.ANVIL
     }
 
+    @JvmStatic
```

Comment:
why is this JvmStatic?

Reply:
all the other methods are too i thought id keep the pattern :D i dont mind accessing it with .INSTANCE in java . change it if you want